.claude/agent-memory/game-archaeology-specialist/mercury-cipher-chain.md

```
@@ -5,6 +5,10 @@ metadata:
   type: project
 ---
 
+> [!NOTE] PROMOTION TARGET: spec.protocol.cipher-and-auth §"PacketEncrypter object layout" + spec.protocol.mercury-wire-format §"cipher envelope"
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)